demo/src/screens/MainScreen.js

```diff
@@ -413,19 +413,19 @@ const SectionItem = React.memo(props => {
       activeOpacity={1}
       style={Dividers.d10}
     >
-      <Text style={[item.deprecate && styles.entryTextDeprecated]} grey10 text80>
+      <Text style={[item.deprecate && styles.entryTextDeprecated]} text80>
         {item.title}
       </Text>
-      <Icon source={chevronIcon} style={{tintColor: Colors.grey10}} supportRTL/>
+      <Icon source={chevronIcon} style={{tintColor: Colors.$iconDefault}} supportRTL/>
```

Comment:
No need to set the default token, it's already set be default (:

Reply:
Without the tintColor prop set to iconDefault token in dark mode I don't see the icon.

Reply:
Ok, I understand why. 
It's because we set the default color in private and not public.